### PR TITLE
 Added Character perturbations along with a test

### PR DESCRIPTION
**Pull request by blackrosedragon2**

### CODE REVIEW

File: tests/test_perturb.py [hunk 1]
```diff
@@ -51,17 +51,22 @@ def test_perturb_insert_space_with_character_size_less_than_two():
         space_perturb.apply("H", ignore=False)
```
Comment (rajaswa) [inline tests/test_perturb.py:51]: Handle already existing test cases

File: tests/test_perturb.py [hunk 2]
```diff
@@ -51,17 +51,22 @@ def test_perturb_insert_space_with_character_size_less_than_two():
...
 
 
-@pytest.mark.parametrize(
-    "word, expected_result", [("THAT", "TAHT"),],
-)
+@pytest.mark.parametrize("word, expected_result", [("hello", "hellzo")])
+def test_perturb_insert_character(word, expected_result):
+    random.seed(30)
+    char_perturb = perturbations.SpaceCharacterPerturbations()
```
[the remark this answers is not in the document]
Comment (rajaswa) [reply tests/test_perturb.py:57]: @blackrosedragon2 change class name in tests also

**Comment by rajaswa** [comment]
@blackrosedragon2  there is an issue with black formatting for the tests
https://pypi.org/project/black/ Run this in the repository root directory
```
black .
```
This shall solve all the issues

**Comment by blackrosedragon2** [comment]
Thank you @someshsingh22

**Pull request merged**